evm-contracts/src/v0.8/dev/ValidatorProxy.sol

```diff
@@ -238,14 +265,30 @@ contract ValidatorProxy is AggregatorValidatorInterface, ConfirmedOwner {
     external
     view
     returns(
-      address current,
+      AggregatorValidatorInterface current,
       bool hasProposal,
-      address proposed
+      AggregatorValidatorInterface proposed
     )
   {
     current = s_currentValidator.target;
     hasProposal = s_currentValidator.hasNewProposal;
     proposed = s_proposedValidator;
   }
 
-}
\ No newline at end of file
+  /**
+   * @notice The type and version of this contract
+   * @return Type and version string
+   */
+  function typeAndVersion()
+    external
+    pure
+    virtual
+    override
+    returns (
+      string memory
+    )
+  {
+    return "ValidatorProxy 1.0.0";
```

Comment:
Let's please leave this as `ValidatorProxy 1.0.0-alpha` until the contract is actually finalized. I want to make sure we don't deploy development versions of the contract with a version suggesting they're the production contract.